pkg/console/operator/operator.go

```diff
@@ -250,6 +250,7 @@ func (c *consoleOperator) defaultConsoleOperatorConfig() *operatorsv1.Console {
 		Spec: operatorsv1.ConsoleSpec{
 			OperatorSpec: operatorsv1.OperatorSpec{
 				ManagementState: operatorsv1.Managed,
+				LogLevel:        operatorsv1.Debug,
```

Comment:
The log-level is set here

Reply:
So the `- "-v=4"` there is for the `operator` pod.

We want to pipe logging down into the `console` pod.

Reply:
So we will need to pipe the log level here:

https://github.com/openshift/console-operator/blob/master/pkg/console/subresource/deployment/deployment.go#L204

Which I imagine will require a bit of a translation function from `operatorsv1.Debug` to whatever `capnslog` wants for `Debug`.  They may be the same, maybe not.  But probably will be some kind of `--log-level=*=DEBUG` string creation.

Reply:
OK this should work, though should we set it to `NORMAL` rather than `DEBUG`?

Reply:
yep, just wanted to see what it will spit out :)

Reply:
@benjaminapetersen updated

Reply:
Sorry meant `Normal` should be the default right?
Even though the `console` `deployment` itself is really sparse on logs...